Readme.md

```diff
@@ -4,17 +4,43 @@ Source for https://api.ooni.io/
 
 ## Local development
 
-To run this locally you need `docker`, `docker-compose` and `make`.
+### Requirements
 
-If those are installed you will then be able to run the following:
+* Docker
+* Docker Compose
+* Make
+* Python >= 3.5
+* Postgresql
 
-```
-make serve
+### Quickstart
+
+**Note**: the default database configuration is `postgres@localhost:5432/ooni_measurements`,
+you only need to run the second step below in case you want to use a different one.
```

Comment:
What is the second step? `export DATABASE_URL`? I'd suggest to say that explicitly.

Reply:
Make sense, gonna change it!

Reply:
@bassosimone Added it into the paragraph, see if it's any better now

Reply:
thanks